spark/src/main/scala/geotrellis/spark/reproject/TileRDDReproject.scala

```diff
@@ -81,6 +82,76 @@ object TileRDDReproject {
           }
       }
 
+    val (sampleKey, BufferedTile(sampleTile, sampleGridBounds)) = bufferedTiles.first() // XXX
```

Comment:
In the `TileRDDReproject` tests, `metadata.layer.tile(Cols|Rows)` reports 50x50 which is not close enough to the actual sizes for those tests to pass.

At present, I still need `sampleTile` and `sampleGridBounds`.